Source/ACE/Entity/Landblock.cs

```diff
@@ -509,6 +510,25 @@ public void EnqueueBroadcast(Position pos, float distance, params GameMessage[]
         }
 
         /// <summary>
+        /// Enqueues a message for global broadcast, thread safe
+        /// </summary>
+        /// <param name="msg"></param>
+        public void EnqueueGlobalBroadcast(params GameMessage[] msgs)
+        {
+            // Atomically checks and sets the broadcastQueued bit --
+            //    guarantees that if we need a broadcast it will be enqueued in the world-managers broadcast queue exactly once
+            if (Interlocked.CompareExchange(ref broadcastQueued, 1, 0) == 0)
+            {
+                WorldManager.BroadcastQueue.EnqueueAction(new ActionEventDelegate(() => SendGlobalBroadcasts()));
```

Comment:
@ddevec i don't follow, probably because I'm out of my depth. Feel free to pull the PR and make any changes needed. The goal is that a object in world can trigger a global broadcast. I probably should move the handlers to WorldObject or maybe Creature.. Not sure if there were any objects outside of the Creature class that could generate a world broadcast.. such as when you kill something or enlighten, or in the case of an admin, issue a global message